project/Dependencies.scala

```diff
@@ -97,6 +99,8 @@ object Dependencies {
     scalaTest,
 
     slick,
+    slickPg,
+    slickPgSpray,
```

Comment:
I think we were saying we want to switch to Circe but Brian said he had some reservations. We can always change this to work with Circe once we decide.

Reply:
Brian's other PRs use circe... we should definitely standardize on 1 json library, and circe >>> spray-json IMHO :)

Reply:
Ah, I need to clarify. My reservation isn't with circe itself but with automatic derivation. We can still choose semi-automatic derivation or go with fully manual.